src/content-layout/internal.tsx

```diff
@@ -26,19 +26,15 @@ export default function InternalContentLayout({
   const isVisualRefresh = useVisualRefresh();
   const overlapElement = useDynamicOverlap();
 
-  /**
-   * Disable the overlap if the component is missing either a header or child
-   * content. If the component is not using visual refresh then the overlap
-   * will not be displayed at all. This is handled in the CSS not the JavaScript.
-   */
-  const isOverlapDisabled = !children || !header || disableOverlap;
+  const isOverlapDisabled = !children || disableOverlap;
 
   return (
     <div
       {...baseProps}
       className={clsx(baseProps.className, styles.layout, {
         [styles['is-overlap-disabled']]: isOverlapDisabled,
         [styles['is-visual-refresh']]: isVisualRefresh,
+        [styles['increased-overlap']]: isVisualRefresh && !header,
```

Comment:
This is a nit pick that you can ignore, but if we are already passing in `is-visual-refresh` you can skip the compute and pass `has-header`. Then the CSS has the values it needs and can compute within i.e. `.is-visual-refresh:not(.has-header)`.